incubator/azuremonitor-containers/templates/omsagent-rbac.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
```

Comment:
This should be created only if a value is set. You cannot assume all clusters have or want rbac resources created.

Reply:
Fixed it. Thx for pointing out.